### DISCUSSION TITLE
Is there a shortcode to display meetings by type

### DISCUSSION THREAD

**Discussion by district6wc** [code4recovery/12-step-meeting-list#1481] [Q&A]
Hi

I'd like to list the Alateen meetings on our Alateen page, is there a shortcode that can manage this?

Thanks
Rose L

**Comment by ginslo** [comment]
Hi @district6wc,  as described in the docs...

Yes, you can use `[tsml_meeting_count]`, `[tsml_location_count]`, `[tsml_group_count]`, and `[tsml_region_count]` to display human-formatted counts of your entities. For example, “Our area currently comprises `[tsml_meeting_count]` meetings.”

`[tsml_next_meetings count="5"]` displays a small table with the next several meetings in it. Use the `count` parameter to adjust how many are displayed. This will be unstyled if you're not using bootstrap in your theme.

Use `[tsml_types_list]` and `[tsml_regions_list]` to output linked lists to your meeting finder.

Use `[tsml_ui]` to display the TSML UI meeting finder.

**Reply by k8fish3r** [reply]
Can I be nosey and ask any examples of how this has been used on other sites?

Thank you for sharing shortcodes, I didn't realise this functionality existed. (RTFM) and remember what I read... I did not :)

**Comment by district6wc** [comment]
Thanks will try this

**Comment by district6wc** [comment]
Thanks for all this info, I was hoping to do a calendar of just the Alateen meetings, I guess there is no way to do that right now. Can I make a suggestion for this to be a feature request so we could have just specific types of meeting listed in calendar format?

**Comment by district6wc** [comment]
We use the short code in our sidebar on all pages so there is always a
meeting preview of the upcoming meetings
I was hoping to have an option for displaying types of meetings
specifically for our Alateen page so that we just show the Alateen type
meeting.

Thank you.

Yours in Service,
Rose Lanaro
Saturday Morning Live, Chapel Hill

District 6 Website Coordinator Panel 62
631-525-3328

**Reply by joshreisner** [reply]
hi rose, thanks for the extra info! is this the page in question? https://alanonalateen6nc.org/alateen-coordinator-contact/

if so, it sounds like you are looking to be able to pass one or more types to the `tsml_next_meetings` shortcode

**Comment by district6wc** [comment]
Josh,

Yes, for that page I wanted to create an instance of the meeting list in
the body of the page that would just list the Alateen meetings.

Thank you.

Yours in Service,
Rose Lanaro
Saturday Morning Live, Chapel Hill

District 6 Website Coordinator Panel 62
631-525-3328
